tests/test_subscription.py

```diff
@@ -668,3 +659,26 @@ def test_sync_metered_plan(
         self.assert_fks(
             subscription, expected_blank_fks=self.default_expected_blank_fks
         )
+
+    @patch("stripe.Plan.retrieve", return_value=deepcopy(FAKE_PLAN), autospec=True)
+    @patch(
+        "stripe.Product.retrieve", return_value=deepcopy(FAKE_PRODUCT), autospec=True
+    )
+    @patch(
+        "stripe.Subscription.retrieve",
+        return_value=deepcopy(FAKE_SUBSCRIPTION_TRIAL_END_NOW),
+        autospec=True,
+    )
+    @patch(
+        "stripe.Customer.retrieve", return_value=deepcopy(FAKE_CUSTOMER), autospec=True
+    )
+    def test_trial_ends_now(
+        self,
+        customer_retrieve_mock,
+        subscription_retrieve_mock,
+        product_retrieve_mock,
+        plan_retrieve_mock,
+    ):
+        subscription_fake = deepcopy(FAKE_SUBSCRIPTION_TRIAL_END_NOW)
+        subscription = Subscription.sync_from_stripe_data(subscription_fake)
+        self.assertEqual(subscription.trial_end.date(),timezone.datetime.now().date())
```

Comment:
I suspect there's a timezone bug here  - maybe better to check for a small delta vs utc now instead?  Or use `timezone.localdate()`, which I think should match the tz used by the model fields.